sdks/python/apache_beam/examples/inference/README.md

```diff
@@ -60,27 +61,28 @@ for details."
 
-The pipeline reads the images, performs basic preprocessing, passes them to the PyTorch implementation of RunInference, and then writes the predictions to a text file.
+The pipeline reads the images, performs basic preprocessing, passes the images to the PyTorch implementation of RunInference, and then writes the predictions to a text file.
 
 ### Dataset and model for image classification
 
-You will need to create or download images, and place them into your `IMAGES_DIR` directory. One popular dataset is from [ImageNet](https://www.image-net.org/). Please follow their instructions to download the images.
-- **Required**: A path to a file called `IMAGE_FILE_NAMES` that contains the absolute paths of each of the images in `IMAGES_DIR` on which you want to run image segmentation. Paths can be different types of URIs such as your local file system, a AWS S3 bucket or GCP Cloud Storage bucket. For example:
+Create a directory named `IMAGES_DIR`. Create or download images and put them in this directory. The directory is not required if image names in the input file `IMAGE_FILE_NAMES` (see below) have absolute paths.
+One popular dataset is from [ImageNet](https://www.image-net.org/). Follow their instructions to download the images.
+- **Required**: A path to a file named `IMAGE_FILE_NAMES` that contains the absolute paths of each of the images in `IMAGES_DIR` that you want to use to run image classification. Paths can be different types of URIs such as your local file system, an AWS S3 bucket, or a GCP Cloud Storage bucket. For example:
 ```
 /absolute/path/to/image1.jpg
 /absolute/path/to/image2.jpg
 ```
-- **Required**: A path to a file called `MODEL_STATE_DICT` that contains the saved parameters of the maskrcnn_resnet50_fpn model. You will need to download the [mobilenet_v2](https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv2.html) model from Pytorch's repository of pretrained models. Note that this requires `torchvision` library.
+- **Required**: Download the [mobilenet_v2](https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv2.html) model from Pytorch's repository of pretrained models. This model requires the torchvision library. To download this model, run the following commands:
 ```
 import torch
 from torchvision.models.detection import mobilenet_v2
 model = mobilenet_v2(pretrained=True)
 torch.save(model.state_dict(), 'mobilenet_v2.pth')
 ```
-- **Required**: A path to a file called `OUTPUT`, to which the pipeline will write the predictions.
-- **Optional**: `IMAGES_DIR`, which is the path to the directory where images are stored. Not required if image names in the input file `IMAGE_FILE_NAMES` have absolute paths.
+- **Required**: A path to a file named `MODEL_STATE_DICT` that contains the saved parameters of the `mobilenet_v2` model.
```

Comment:
Could we instead consolidate the above bullet with this one? That way each bullet corresponds to one of the inputs in "Running `pytorch_image_classification.py`." Something like this:

```suggestion
- **Required**: A path to a file named `MODEL_STATE_DICT` that contains the saved parameters of the `mobilenet_v2` model. You can download the mobilenet_v2 model from ... use the following script:
```

Reply:
I broke these into two steps and changed the order so that users would see them in the order they would need to complete the steps. Presumably they download the model and then create the parameters. In general in documentation, it's a best practice for each step to only be one action.

Reply:
That makes sense, but to me this looks like a bulleted list defining the four inputs to the script, not a list of directions/actions. For each input it identifies if they are Required/Optional, and explains how to populate them. It's odd to me to then have one separate bullet that represents an instruction.

Reply:
You're right. I halfway made these steps. I'll update them to be steps instead of a list of requirements, because steps will be easier for users to follow.